commercial/app/controllers/BookOffersController.scala

```diff
@@ -78,6 +78,36 @@ class BookOffersController(bookFinder: BookFinder, bestsellersAgent: Bestsellers
     booksSample(specificIds, segment) map result
   }
 
+  def getBook = Action.async { implicit request =>
+    specificId map { isbn =>
+      bookFinder.findByIsbn(isbn) map {
+        _ map { book =>
+          val json = Json.toJson(book)
+          Cached(60.seconds){
```

Comment:
It looks like there's a value here: https://github.com/guardian/frontend/blob/master/commercial/app/controllers/package.scala#L12